client/src/app/project-create-edit/components/project-metadata-form/project-metadata-form.component.ts

```diff
@@ -171,16 +174,30 @@ export class ProjectMetadataFormComponent implements OnInit, OnDestroy {
     if (this.project?.content) {
       if (this.project?.content.hasOwnProperty('describedBy')
         && this.project?.content.hasOwnProperty('schema_type')) {
-        this.schema = this.project.content['describedBy'];
+        this.schemaUrl = this.project.content['describedBy'];
+        this.schemaService.getDereferencedSchema(this.schemaUrl)
```

Comment:
This code is duplicated on line 197-199. Do we really need this here? Tx

Reply:
yes, it's needed. I refactored it to avoid duplication. Please review. Thanks!